spacy/tests/lang/nl/test_noun_chunks.py

```diff
@@ -207,3 +208,18 @@ def test_chunking(nl_sample, nl_reference_chunking):
     """
     chunks = [s.text.lower() for s in nl_sample.noun_chunks]
     assert chunks == nl_reference_chunking
+
+
+@pytest.mark.issue(10846)
+def test_no_overlapping_chunks(nl_vocab):
+    # fmt: off
+    doc = Doc(
+        nl_vocab,
+        words=["okt", "opvlamming", "panuveitiss", "Oculo", "sinistra", ">", "Oculo", "dextra", "en", "alghele", "malaise", "moe", ",", "tattoos", "opgezet", ".", "17-03-2020", "Ozurdex", "Oculo", "dextra", "et", "sinistra", ",", "opvlamming", "onder", "humir", "luchtweg", "ondanks", "wekelijks", "ada", "vele", "corpus", "vitreum", "(", "glasvocht", ")", "troebelingen", "beschreven"],
+        deps=["parataxis", "ROOT", "ROOT", "flat", "flat", "flat", "nsubj", "appos", "cc", "conj", "obj", "ROOT", "punct", "obj", "conj", "punct", "nummod", "ROOT", "nmod:poss", "appos", "flat", "flat", "punct", "appos", "case", "nmod", "ROOT", "case", "advmod", "nmod", "amod", "nsubj", "nmod", "punct", "nmod", "punct", "nsubj:pass", "ROOT"],
+        heads=[1, 1, 2, 2, 2, 2, 11, 6, 9, 7, 11, 11, 14, 14, 11, 11, 17, 17, 26, 18, 18, 18, 23, 18, 25, 23, 26, 32, 29, 32, 31, 29, 26, 34, 32, 34, 37, 37],
+        pos=["NOUN", "NOUN", "PROPN", "PROPN", "ADJ", "PUNCT", "PROPN", "NOUN", "CCONJ", "ADJ", "NOUN", "ADJ", "PUNCT", "ADJ", "VERB", "PUNCT", "NUM", "NOUN", "NOUN", "VERB", "PROPN", "PROPN", "PUNCT", "NOUN", "ADP", "PROPN", "NOUN", "ADP", "ADJ", "PROPN", "ADV", "NOUN", "NOUN", "PUNCT", "NOUN", "PUNCT", "NOUN", "VERB", ],
+    )
```

Comment:
This example is barely Dutch and also a tad long for the test suite.

This one has the same issue (3 noun chunks before this PR including a nested one, 2 after this PR):

```suggestion
    doc = Doc(
        nl_vocab,
        words=["Dit", "programma", "wordt", "beschouwd", "als", "'s", "werelds", "eerste", "computerprogramma"],
        deps=["det", "nsubj:pass", "aux:pass", "ROOT", "mark", "det", "fixed", "amod", "xcomp"],
        heads=[1, 3, 3, 3, 8, 8, 5, 8, 3],
        pos=["DET", "NOUN", "AUX", "VERB", "SCONJ", "DET", "NOUN", "ADJ", "NOUN"],
    )
```